plaidml/bridge/openvino/src/plaidml_plugin/ops/tile.cpp

```diff
@@ -0,0 +1,27 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "plaidml_ops.hpp"
+#include "plaidml_util.hpp"
+
+#include "ngraph/opsets/opset.hpp"
+#include "ngraph/opsets/opset1.hpp"
+
+#include "plaidml/op/op.h"
+
+using namespace plaidml;          // NOLINT[build/namespaces]
+using namespace InferenceEngine;  // NOLINT[build/namespaces]
+
+namespace PlaidMLPlugin {
+
+static OpRegistration reg("tile", [](const Context& ctx) {
+  IE_ASSERT(ctx.operands.size() == 2);
+  auto I = ctx.operands.at(0);
+  auto reps = get_shape_from_constant_operand(1, ctx.layer);
+  std::vector<int> reps_ints;
+  reps_ints.assign(reps.begin(), reps.end());
```

Comment:
The `reps_ints` is here because you need the vector to use a different type of integer than the one returned from `get_shape_from_constant_operand`, correct? We should upgrade the oplib to be nicer about this at some point, but no need to do that in this PR.

Reply:
yes.